api/src/operations/notification/index.ts

```diff
@@ -0,0 +1,28 @@
+import { defineOperationApi } from '@directus/shared/utils';
+import { NotificationsService } from '../../services';
+
+type Options = {
+	recipient: string;
+	subject: string;
+	message: string | null;
+};
+
+export default defineOperationApi<Options>({
+	id: 'notification',
+
+	handler: async ({ recipient, subject, message }, { accountability, database, getSchema }) => {
+		const notificationsService = new NotificationsService({
+			schema: await getSchema({ database }),
+			accountability,
+			knex: database,
+		});
+
+		const result = await notificationsService.createOne({
+			recipient,
+			subject,
+			message,
+		});
+
+		return result;
+	},
+});
```

Comment:
I could not get a notification to fire.
Can you let me know what the syntax is to define a user?
When I try to type @some-user-name the name doesn't auto-fill.
When I manually type in a name or email, nothing happens.

Reply:
Try using the users uuid instead.

Reply:
You mean type it in directly? I tried here, but it didn't work either.
The big question I have is what are we basing al the command syntax off of?
I notice we use dot-notation. Several areas where it looks like we can input raw data.
I just don't know what variables and objects that Data Flows has access to.

https://user-images.githubusercontent.com/62306630/167666595-dcd93d09-6a28-4068-b078-106783c7a4a0.mov